Mac/Components/Shared/PlayerStateInfoController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  MediaInfoController.swift
+//  Persephone
+//
+//  Created by Alan Harper on 18/11/20.
+//  Copyright © 2020 Dan Barber. All rights reserved.
+//
+
+import Foundation
+import ReSwift
+import MediaPlayer
+import Kingfisher
+
+class PlayerStateInfoController {
+  init() {
+    App.store.subscribe(self) {
```

Comment:
We subscribe to the current song separately from the play/pause status so we don't continually update the current album, including fetching artwork